src/packages/string_more.rs

```diff
@@ -84,8 +84,7 @@ mod string_functions {
     }
 
     #[rhai_fn(name = "+")]
-    pub fn add_append_unit(string: ImmutableString, item: ()) -> ImmutableString {
-        let _ = item;
+    pub fn add_append_unit(string: ImmutableString, _item: ()) -> ImmutableString {
```

Comment:
Again, let's not rename parameters with underscores but put `#[allow(unused_variables)]` like in `string_basic.rs`...